internal/usage/usage.go

```diff
@@ -34,7 +34,8 @@ const (
 	NExamples                = "Maximum number of example queries to provide that will be improved by a suggested index."
 	NIndexes                 = "Maximum number of indexes to suggest."
 	MDBVersion               = "Major MongoDB version of the cluster."
-	AuthDB                   = "Authentication database name."
+	AuthDB                   = "Authentication database name. If the user authenticates with AWS IAM, x.509, or LDAP, this value should be $external. If the user authenticates with SCRAM-SHA, this value should be admin."
+	MCLIAuthDB               = "Authentication database name."
```

Comment:
it would've been easier to create `AtlasAuthDB` but if this is the way you want to play it please update to sue this new variable

https://github.com/mongodb/mongodb-atlas-cli/blob/73bd6d30c60771c02af52599b644aafaf323312f/internal/cli/opsmanager/dbusers/create.go#L131

Reply:
@gssbzn That makes sense, I can change it to do an Atlas variable as the exception. Right now I'm trying to figure out if I can make some examples only show up for Atlas CLI, so this PR may take a little time while I figure that out